route-recognizer/route-recognizer.d.ts

```diff
@@ -4,22 +4,21 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare module "route-recognizer" {
-	
-	class RouteRecognizer<H> {
+
+	export default class RouteRecognizer<H> {
```

Comment:
Ahh yes. It uses esnext which was deprecated before cross compilation was agreed upon (eg. `.default`). Does TS not resolve `module.exports.default` to `modules.exports` if the former is not present?